tests/transactions.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest');
+const baseUrl = 'localhost:8000';
+const { v4: uuidv4 } = require('uuid');
+
+test('airtime transfer function', async () => {
+  const reference = uuidv4();
+
+  const mockrequest = {
+    country: 'NG',
+    customer: '+23490803840303',
+    amount: 100,
+    recurrence: 'ONCE',
+    type: 'AIRTIME',
+    reference,
+  };
+
+  const response = await request(baseUrl).post('/api/v1/transfer/airtime').send(mockrequest);
+
+  // the status code is going to be 500 because  the wallet has insufficient funds
+  expect(response.statusCode).toBe(500);
+
+  expect(response.body.error).not.toBe(null);
+});
+
+test('data transfer function', async () => {
+  const reference = uuidv4();
+
+  const mockrequest = {
+    country: 'NG',
+    customer: '+23490803840303',
+    amount: 100,
+    type: 'MTN 50 MB',
+    recurrence: 'ONCE',
+    reference,
+  };
+  const response = await request(baseUrl).post('/api/v1/transfer/data').send(mockrequest);
+
+  // // the status code is going to be 500 because  the wallet has insufficient funds
+
```

Comment:
How does the server know the wallet funds are insufficient when they did not send any wallet id? I'm confused

Reply:
That is the normal response we get when ever we use those endpoints because we haven't set up the wallet tracking and funding features.